Steamworks/src/org/usfirst/frc/team4786/robot/commands/DriveToPosition.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team4786.robot.commands;
+
+import org.usfirst.frc.team4786.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class DriveToPosition extends Command {
+
+	private double targetPosition;
+	
+    public DriveToPosition(double distance) {
+    	//We require the driveTrain to drive, obviously
+    	requires(Robot.driveTrain);
+    	
+    	//So we can use this variable in the execute() function
+    	targetPosition = distance;
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	//Let's drive!
+    	Robot.driveTrain.driveToPosition(targetPosition);
```

Comment:
One way to make this work a little better (and to be able to use this after moving without this command, you can move this function call to the `initialize()` method).